style.css

```
@@ -0,0 +1,135 @@
+:root {
+  --font-family-heading: "Anton", sans-serif;
+  --font-famiky-body: "Noto Sans JP", sans-serif;
```

```suggestion
  --font-family-body: "Noto Sans JP", sans-serif;
```
VS Codeの拡張機能である「Code Spell Checker」を使用するとスペルミスに気づきやすくなるので、インストールしておくのをおすすめします！👍

---

修正しました！
拡張機能の「Code Spell Checker」もインストールしました！